__tests__/ship.test.js

```diff
@@ -0,0 +1,100 @@
+/* globals describe it expect */
+const Ship = require('../src/ship');
+const Port = require('../src/port');
+const Itinerary = require('../src/itinerary');
+
+describe('Ship constructor', () => {
+  describe('Ships with ports and an itinerary', () => {
+    let ship;
+    let dover;
+    let calais;
+    let itinerary;
+    // let port;
```

Comment:
Why is this commented out? It shouldnt be as you are modifying it in your beforeEach block
```
    let port;
```